db_stress_tool/db_stress_test_base.cc

```diff
@@ -1681,6 +1703,35 @@ Status StressTest::TestBackupRestore(
   } else {
     backup_opts.schema_version = 2;
   }
+  if (thread->rand.OneIn(3)) {
+    backup_opts.max_background_operations = 16;
+  } else {
+    backup_opts.max_background_operations = 1;
+  }
+  if (thread->rand.OneIn(2)) {
+    backup_opts.backup_rate_limiter.reset(NewGenericRateLimiter(
+        1024 * 1024 /* rate_bytes_per_sec */, 10 /* refill_period_us */));
+  }
+  if (thread->rand.OneIn(2)) {
+    backup_opts.restore_rate_limiter.reset(NewGenericRateLimiter(
+        1024 * 1024 /* rate_bytes_per_sec */, 10 /* refill_period_us */));
+  }
```

Comment:
3/4 of the time there's a 1MB/s rate limit in the backup+restore path. Will all the threads eventually get stuck on backups?

Reply:
Fixed to refilling of FLAGS_backup_max_size per us (refill period)